packages/synapse-react-client/src/components/CreateOrUpdateAccessRequirementWizard/CreateOrUpdateAccessRequirementWizard.integration.test.tsx

```diff
@@ -75,7 +74,6 @@ function setUp(props: CreateOrUpdateAccessRequirementWizardProps) {
 
 async function expectNoContentWiki() {
   await screen.findByRole('button', { name: 'Edit Instructions' })
-  expectMarkdownSynapseNotToGetWiki()
```

Comment:
Remove/modify many test assertions that depended on the implementation details of MarkdownSynapse